### DISCUSSION TITLE
SLEAP experiment

### DISCUSSION THREAD

**Discussion by D19CrMei** [talmolab/sleap#819] [Help!]
Hello, I would like to ask three questions about the SLEAP experiment. 
1. In Bottom-up, I set the animal skeleton to be non-tree with three independent edges. The GUI sometimes says it can be trained, sometimes it can't.
2. In the case of occasional training, the output results only have one side, and none else. What is the reason for this?
3. The results of the training show "not data in train" in part. What is the situation? It appears between two nodes

**Comment by roomrys** [comment]
Hi @D19CrMei,

Do you mind providing some screenshots/error logs of the issues you are experiencing?

1. Bottom up model need to have a tree graph for the skeleton as per this [comment](https://github.com/talmolab/sleap/discussions/820#discussioncomment-3050065).
2. Can you provide some screenshots please
3. Can you provide some screenshots please

Thanks,
Liezl
